examples/Hermitian_Phasing.py

```diff
@@ -30,21 +30,6 @@
             "vd",
             "IR",
         ),
-        (
-            (
-                23
-                * (1 - (32 * power / (0.25 + power)) * 150e-6 * 659.33)
-                * s.exp(+1j * 2 * s.pi * 100 * (t2) - abs(t2) * 50 * s.pi)
-            ),
-            [
-                ("power", nddata(r_[0:4:25j], "power")),
-                ("ph1", nddata(r_[0:4] / 4.0, "ph1")),
-                ("t2", nddata(r_[0:0.2:256j] - echo_time, "t2")),
-            ],
-            {"ph1": 1},
-            "power",
-            "enhancement",
-        ),
```

Comment:
this actually failed -- you want to ad in an example that does an enhancement curve, but this was actually not run b/c of the quit statement that you had at the end!

Reply:
so without the quit statement it runs on my end!